src/traits.rs

```diff
@@ -733,6 +733,20 @@ pub trait SystemExt: Sized + Debug + Default {
     /// ```
     fn get_processors(&self) -> &[Processor];
 
+    /// Returns the physical numer of cores on the processor.
+    ///
+    /// In case there are multiple CPUs, it will combine the physical core count of all the CPUs.
+    ///
+    /// **Important**: this information is computed every time this function is called.
+    ///
+    /// ```no_run
+    /// use sysinfo::{ProcessorExt, System, SystemExt};
+    ///
+    /// let s = System::new();
+    /// println!("{}", s.get_physical_core_numbers());
+    /// ```
+    fn get_physical_core_numbers(&self) -> usize;
```

Comment:
What do you think about `get_physical_core_count` instead?

Reply:
I think both are good but as `get_physical_core_count` is more likely to be written out in a sentence I think this one is better.  
Also in the documentation about this function we use count.  
Will change.